### ISSUE TITLE
All wheels for 3.4.1 gone forever on PyPI

### ISSUE THREAD

**Issue by tomkooij** [PyTables/PyTables#621]
TL;DR: I deleted the wheels for 3.4.1 from PyPI. I cannot reupload them (not even the original versions). I need to issue v3.4.2.

I build new wheels for 3.4.1 (wheels.scipy.org):
 - manylinux py34 wheels
 - OSX wheels
 - manylinux wheels with the correct blosc 

I tried uploading them to PyPI, but that was refused.  So I deleted the wheels.

Now I cannot reupload them. **NOT EVEN THE ORIGINAL WHEELS**. There are many issues about this already reported on PyPI. It's a feature, not a bug.

Anyway, now I have to bump the version in master and build new wheels... I'll probably do that tonight.

In the meantime: There are no wheels on PyPI.

**Comment by tomkooij** [comment]
There does seems to be a way to fix this using build tags:
https://github.com/MacPython/wiki/wiki/Build-Tags

(But wheels for release 3.4.2 are building...)

**Comment by tomkooij** [comment]
Wheels for 3.4.1 are back on PyPI (thanks to the build tag).

I'll upload 3.4.2 to PyPI tomorrow when new wheels are built.

**Comment by andreabedini** [comment]
> Now I cannot reupload them. NOT EVEN THE ORIGINAL WHEELS. There are many issues about this already reported on PyPI. It's a feature, not a bug.

happened to me once already 🤗
